Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -133,6 +138,20 @@ private extension ReceiptRenderer {
 
         return summaryContent
     }
+
+    private func regulatoryItems() -> String {
+        guard let emv = parameters.cardDetails.receipt else {
+            return "<br/>"
+        }
+
+        /// According to the documentation, only `Application name` and `AID`
+        /// are required in the US.
+        /// https://stripe.com/docs/terminal/checkout/receipts#custom
+        return """
+            \(Localization.applicationName): \(emv.applicationPreferredName)<br/>
```

Comment:
What prevents the applicationPreferredName or dedicatedFileName from containing html tags and/or < or > ? We should probably be filtering these strings to strip tags and brackets.

Reply:
Should we strip them or scape them? I mean, if we remove tags and brackets we would be changing the actual content of the string... I've logged a ticket to look into this in more detail: #4137